src/main/java/me/xmrvizzy/skyblocker/config/SkyblockerConfig.java

```diff
@@ -269,6 +269,8 @@ public static class Dungeons {
         public boolean solveThreeWeirdos = true;
         public boolean blazesolver = true;
         public boolean solveTrivia = true;
+        public boolean lividColor = true;
+        public String lividColorText = "[color]";
```

Comment:
This default value was a bit confusing at first until I read code and saw it in action

Reply:
Changed the default text.